Tests/Functional/app/AppKernel.php

```diff
@@ -37,12 +41,12 @@ public function __construct($environment, $debug, $testCase = null)
      */
     public function registerBundles(): array
     {
-        return [
-            new \Symfony\Bundle\FrameworkBundle\FrameworkBundle(),
-            new \Symfony\Bundle\SecurityBundle\SecurityBundle(),
-            new \Lexik\Bundle\JWTAuthenticationBundle\LexikJWTAuthenticationBundle(),
-            new \Lexik\Bundle\JWTAuthenticationBundle\Tests\Functional\Bundle\Bundle(),
-        ];
+        return array(
+            new FrameworkBundle(),
+            new SecurityBundle(),
+            new LexikJWTAuthenticationBundle(),
+            new Bundle(),
+        );
```

Comment:
```suggestion
        return [
            new FrameworkBundle(),
            new SecurityBundle(),
            new LexikJWTAuthenticationBundle(),
            new Bundle(),
        ];
```

Reply:
Strange... Rector reverted to to the old style `array(` for no reason.
It is now fixed.